libs/sdk-backend-spi/src/workspace/elements/index.ts

```diff
@@ -55,6 +55,24 @@ export interface IElementQueryOptions {
     // afm?: GdcExecuteAFM.IAfm; // TODO: do we really need this? if so, we should add support for using executionDefinition here
 }
 
+/**
+ * Attribute filter limiting the elements. To be able to filter elements, the current attribute
+ * and the filter attribute must be connected in the data model. The property "overAttribute" identifies
+ * the connecting table in the logical data model. For method providing all possible connecting attributes
+ * see "getCommonAttributes"
+ *
+ * @public
+ */
+
+export interface IElementQueryAttributeFilter {
+    attributeFilter: {
+        displayForm: ObjRef;
+        attributeElements: ObjRef[];
+        negativeSelection: boolean;
+    };
```

Comment:
this reminds me of positive & negative attribute filters. I suggest to switch this to an already established type from sdk-model:
```
attributeFilter: IAttributeFilter 
```

seems like a good fit to me. indeed, IAttributeFilter can contain elements by reference or by value. we can start by supporting elements by reference (uris) in this PR. follow will be to translate values to URIs (there's a bulk endpoint for that so should be fairly simple)